spec/policies/hotel_policy_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe HotelPolicy, type: :policy do
+  subject { described_class }
+
+  let(:user) { create(:user) }
+
+  permissions :show? do
+    it 'User can show hotel' do
+      expect(subject).not_to permit(user)
+    end
+
+    it "Owner can't show hotel" do
+      expect(subject).not_to permit(User.new(owner: true))
+    end
+  end
+
+  permissions :create? do
+    it "User can't create hotel" do
+      expect(subject).not_to permit(user)
+    end
+
+    it 'Owner can create hotel' do
+      expect(subject).to permit(User.new(owner: true))
```

Comment:
you just generate users without any factory. You should create 2 users in the main describe and use both of them in tests instead of building a new owner each time